greynoise/constants.py

```diff
@@ -0,0 +1,2 @@
+LOGGER_NAME = "greynoise"
+INTEGRATION_NAME = "fortinet-fortisoar-integration-v1.0.0"
```

Comment:
2.0.0?

Reply:
There is currently a community-contributed 1.0.0 version, and we'd like to be a full replacement of that connector, hence the rev in version number.  If you'd like us to approach this differently, please let me know

Reply:
how does the GreyNoise api use the integration_name input? does it have to be pre-registered? would the auth fail if we specify 2.0.0 in the INTEGRATION_NAME?

Reply:
So, this should actually be 2.0.0, that's a mistake and I've updated it to match the version.  It doesn't need to be pre-registered, it just helps us track usage of the integration on our side.